src/shared/components/case-editor/case-edit-submit/case-edit-submit.html

```diff
@@ -39,7 +39,10 @@ <h2 class="heading-h2">Check your answers</h2>
       <tbody>
       <ng-container *ngFor="let page of this.wizard.pages">
         <ng-container *ngIf="isShown(page)">
-          <ng-container *ngFor="let field of page | ccdPageFields:editForm.controls['data'] | ccdReadFieldsFilter: false :undefined :true :editForm.controls['data']">
+          <ng-container *ngFor="let field of page
+          | ccdPageFields: editForm.controls['data']
+          | ccdReadFieldsFilter: false :undefined :true : editForm.controls['data']
+          | ccdCYAPageLabelFilter ">
```

Comment:
_Nitpick:_
Extraneous space on the end of this line, before the ending quote mark.

Reply:
done